app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    if len(split_command) != 3 or split_command[0] != "mv":
+        print("The format is incorrect")
+        return
+    source, destination = split_command[1], split_command[2]
+    parts = destination.split("/")
```

Comment:
Use the built-in methods to extract required values from the path. You code should be platform-agnostic and it should work with different directories delimiters without hardcoding them